lib/compilers/argument-parsers.js

```diff
@@ -297,6 +297,14 @@ export class RustParser extends BaseParser {
     }
 }
 
+export class RustCGGCCParser extends RustParser {
+    static async parse(compiler) {
+        let scompiler = await RustParser.parse(compiler);
```

Comment:
This can be `const` (JS `const` is for the variable, not the "thing the variable references")

Reply:
ok !